macro/README.md

```diff
@@ -0,0 +1,100 @@
+# Helper macros for Host development 
+
+This crate provides a few helper macros for Host development.
+
+## Usage
+
+Add this to your `Cargo.toml`:
+
+```toml
+[dependencies]
+nexus-macro = { path = "../macro" }
+nexus-profiler = { path = "../macro/profiler" }
+```
+
+## Profile macros
+
+### Requirements
+
+Install `go` programming language.
+
+#### MacOS
+
+```bash
+brew install go
+```
+
+#### Linux
+
+```bash
+sudo apt install golang-go
+```
+
+### How to use `#[profile]` macro
```

Comment:
Maybe be nice to give some examples of usage with the SDK

Reply:
Add example in README.md 

https://github.com/nexus-xyz/nexus-zkvm/commit/b1b02fcafcfa15b5d3ee6c80c55a704117db836c